streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -488,18 +461,7 @@ public void setUncaughtExceptionHandler(final StreamsUncaughtExceptionHandler us
     }
 
     private void defaultStreamsUncaughtExceptionHandler(final Throwable throwable, final boolean skipThreadReplacement) {
```

Comment:
Given that this method just call the other one, it seems we can remove this method, and let the caller call ` handleStreamsUncaughtException` directly?